languages/wikipron/src/postprocess

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eou pipefail
+
+for TSV in ../tsv/*.tsv; do
+    # Explicitly uses byte-wise comparison for sorting
+    # rather than a locale-dependent comparison.
+    LC_ALL=C sort -u -o "${TSV}" "${TSV}"
+    ./generalized_split.py "${TSV}"
+
+done
+./whitelist.py "../tsv/eng_uk_phonemic.tsv"
```

Comment:
Looks good. A few small requests:


* Could you move the blank line on line 9 to after `done` so that it reads roughly

```
...
     ./generalized_split.py "${TSV}"
done

./whitelist.py ../tsv/eng_uk_phonemic.tsv
...
```

* You don't need to quote file names if they don't include spaces, wild-card expansions, or variable expansions, as is the case here.